cdap-security-spi/src/main/java/io/cdap/cdap/security/spi/authorization/AccessControllerAPI.java

```diff
@@ -0,0 +1,10 @@
+package io.cdap.cdap.security.spi.authorization;
+
+/**
+ * Helper interface for API
+ *
+ * This is newer version of {@link AccessController}
+ */
+public interface AccessControllerAPI extends GrantFetcher<Void>, PermissionManager<Void>,
```

Comment:
I don't think we need this one. Anyone using API should just use proper subinterface